packages/components/src/Helpers.purs

```diff
@@ -1,30 +1,32 @@
 module KSF.Helpers where
 
-import           Prelude
+import Prelude
 
-import           Control.Alt ((<|>))
-import           Data.Date               (Date, adjust)
-import           Data.DateTime           (DateTime (..))
-import           Data.DateTime           as DateTime
-import           Data.Enum               (toEnum)
-import           Data.Formatter.DateTime (Formatter, FormatterCommand (..),
-                                          format)
-import           Data.Int                (toNumber)
-import           Data.List               (fromFoldable)
-import           Data.Maybe              (Maybe (..), fromJust)
-import           Data.String             (Pattern (..))
-import           Data.String             as String
-import           Data.Time               (Time (..))
-import           Data.Time.Duration      (Days (..), Hours (..), negateDuration)
-import           Effect                  (Effect)
-import           Effect.Now              as Now
-import           Partial.Unsafe          (unsafePartial)
+import Control.Alt ((<|>))
+import Data.Date (Date, adjust)
+import Data.DateTime (DateTime(..))
+import Data.DateTime as DateTime
+import Data.Enum (toEnum)
+import Data.Formatter.DateTime (Formatter, FormatterCommand(..), format)
+import Data.Int (toNumber)
+import Data.List (fromFoldable)
+import Data.Maybe (Maybe(..), fromJust)
+import Data.String (Pattern(..))
+import Data.String as String
+import Data.Time (Time(..))
+import Data.Time.Duration (Days(..), Hours(..), negateDuration)
+import Effect (Effect)
+import Effect.Now as Now
+import Partial.Unsafe (unsafePartial)
 
 foreign import getCurrentTZOffset_ :: Effect Int
 
 midnight :: Time
 midnight = unsafePartial $ fromJust $ Time <$> toEnum 0 <*> toEnum 0 <*> toEnum 0 <*> toEnum 0
 
+almostMidnight :: Time
```

Comment:
You aren't using this constant in this version after all, right? Looks like there's no need to touch this file at all at this time.